tests/integration/test_connector_versions.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from httpx import HTTPStatusError
+
+from specklepy.core.api.connector_versions import (
+    ConnectorVersion,
+    ConnectorVersions,
+    get_connector_versions,
+    get_latest_version,
+)
+
+# NOTE: the tests in this file are testing against the live releases.speckle.dev server
+# url defined in get_connector_versions.
+
+
+def test_connector_versions():
+    res = get_connector_versions("blender")
+
+    assert isinstance(res, ConnectorVersions)
+    assert res.versions  # Assuming the feed is not empty
+
+
+def test_get_latest_version_throws_no_slug():
+    with pytest.raises(HTTPStatusError):
```

Comment:
make sure to test the status code, to make it a bit more robust. you can store the error into a variable in the with block.

https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions